### PR TITLE
resource_huaweicloud_as_group support enable, disable and support import

### LINKED ISSUE #837: [Feature Request] As Group support pause, resume and support import
Hi there,

It will be very useful if the `as group` supports `pause` and `resume`.
Plus, if it possilble to support `import` in `as group`?

Thanks

Affected Resource(s)
huaweicloud_as_group
References
https://support.huaweicloud.com/api-as/as_06_0106.html

### PR DESCRIPTION
**Pull request by Jason-Zhang9309**
fixes #837 
Test result
```bash
make testacc TEST='./huaweicloud' TESTARGS='-run TestAccASV1Group_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud -v -run TestAccASV1Group_basic -timeout 360m -parallel 4
=== RUN   TestAccASV1Group_basic
=== PAUSE TestAccASV1Group_basic
=== CONT  TestAccASV1Group_basic
--- PASS: TestAccASV1Group_basic (135.62s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud       135.669s
```

### CODE REVIEW

File: docs/resources/as_group.md [hunk 1]
```diff
@@ -189,6 +189,8 @@ The following arguments are supported:
 * `delete_instances` - (Optional, String) Whether to delete the instances in the AS group
     when deleting the AS group. The options are `yes` and `no`.
 
+* `enable` - (Optional, Bool) Whether to enable the AS Group. The options are `true` and `false`.
```
Comment (ShiChangkuo) [inline docs/resources/as_group.md:192]: we should add the description of default value (true).

File: huaweicloud/resource_huaweicloud_as_group_v1.go [hunk 2]
```diff
@@ -517,7 +527,34 @@ func resourceASGroupRead(d *schema.ResourceData, meta interface{}) error {
...
 	d.Set("scaling_group_name", asg.Name)
+	d.Set("vpc_id", asg.VpcID)
 	d.Set("status", asg.Status)
+
+	if asg.Status == "INSERVICE" {
+		d.Set("enable", true)
+	}
+	if asg.Status == "PAUSED" {
+		d.Set("enable", false)
+	}
```
Comment (ShiChangkuo) [inline huaweicloud/resource_huaweicloud_as_group_v1.go:537]: suggest to set `enable`  to false when the status not equal **INSERVICE**

File: docs/resources/as_group.md (hunk 1 above)
Comment (Jason-Zhang9309) [reply docs/resources/as_group.md:192]: fixed

File: huaweicloud/resource_huaweicloud_as_group_v1.go (hunk 2 above)
Comment (Jason-Zhang9309) [reply huaweicloud/resource_huaweicloud_as_group_v1.go:537]: fixed

**Pull request merged**